Samples/iOS-Swift/iOS-Swift/ViewController.swift

```diff
@@ -292,70 +339,91 @@ class ViewController: UIViewController {
-            
+
             dispatchQueue.async {
                 DSNStorage.shared.saveDSN(dsn: dsn)
             }
         } else {
             sender.backgroundColor = UIColor.systemRed
-            
+
             dispatchQueue.async {
                 DSNStorage.shared.deleteDSN()
             }
         }
     }
     
-    @IBAction func resetDSN(_ sender: Any) {
+    @IBAction func resetDSN(_ sender: UIButton) {
+        highlightButton(sender)
         self.dsnTextField.text = AppDelegate.defaultDSN
         self.dsnTextField.backgroundColor = UIColor.systemGreen
-        
+
         dispatchQueue.async {
             DSNStorage.shared.saveDSN(dsn: AppDelegate.defaultDSN)
         }
     }
     
-    @IBAction func showNibController(_ sender: Any) {
+    @IBAction func showNibController(_ sender: UIButton) {
+        highlightButton(sender)
         let nib = NibViewController()
         nib.title = "Nib View Controller"
         navigationController?.pushViewController(nib, animated: false)
     }
     
-    @IBAction func showTableViewController(_ sender: Any) {
+    @IBAction func showTableViewController(_ sender: UIButton) {
+        highlightButton(sender)
         let controller = TableViewController(style: .plain)
         controller.title = "Table View Controller"
         navigationController?.pushViewController(controller, animated: false)
     }
     
-    @IBAction func useCoreData(_ sender: Any) {
+    @IBAction func useCoreData(_ sender: UIButton) {
+        highlightButton(sender)
         let controller = CoreDataViewController()
         controller.title = "CoreData"
         navigationController?.pushViewController(controller, animated: false)
     }
 
-    @IBAction func performanceScenarios(_ sender: Any) {
+    @IBAction func performanceScenarios(_ sender: UIButton) {
+        highlightButton(sender)
         let controller = PerformanceViewController()
         controller.title = "Performance Scenarios"
         navigationController?.pushViewController(controller, animated: false)
     }
 
-    @IBAction func permissions(_ sender: Any) {
+    @IBAction func permissions(_ sender: UIButton) {
+        highlightButton(sender)
         let controller = PermissionsViewController()
         controller.title = "Permissions"
         navigationController?.pushViewController(controller, animated: true)
     }
     
-    @IBAction func flush(_ sender: Any) {
+    @IBAction func flush(_ sender: UIButton) {
+        highlightButton(sender)
         SentrySDK.flush(timeout: 5)
     }
     
-    @IBAction func close(_ sender: Any) {
+    @IBAction func close(_ sender: UIButton) {
+        highlightButton(sender)
         SentrySDK.close()
     }
-    
-    @IBAction func startSDK(_ sender: Any) {
+
+    @IBAction func startSDK(_ sender: UIButton) {
+        highlightButton(sender)
         AppDelegate.startSentry()
     }
+
+    func highlightButton(_ sender: UIButton) {
```

Comment:
I just wanted to ask why you added this. Many users use the sample app as a reference to play around with the SDK. Highlighting the buttons in blue kind of looks weird to me.

Reply:
This was from https://github.com/getsentry/sentry-cocoa/pull/2827. I was trying to view the videos of UI test runs on saucelabs to debug failures, and couldn't tell which buttons were being pressed.

Reply:
I closed that PR and will remove it from this branch. Refactoring the menu into a collection of screens with table views will take care of this better.